src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Button, Link, Flex, Text } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  return (
+    <>
+      <nav aria-labelledby="platformBtn" className={`platform-navigator`}>
+        <Text fontWeight={"bold"}>Choose your framework/language</Text>
+        <Flex alignItems="center">
+          <Button
+            className={`platform-navigator__button`}
+            aria-expanded="false"
+            id="platformBtn"
+            aria-controls="platformNav"
+            onClick={() => {
+              setIsOpen(!isOpen);
+            }}
+            isFullWidth={true}
+            fontWeight={"normal"}
+          >
+            <Flex as="span" alignItems="center">
+              {platformItem.icon}
+              {currentPlatform}
+            </Flex>
+            <IconChevron className={isOpen ? '' : 'icon-rotate-90-reverse'} />
+          </Button>
+          <InfoPopover platform={currentPlatform} />
+        </Flex>
+        <ul
+          id="platformNav"
+          className={`platform-navigator__dropdown ${
+            isOpen ? 'platform-navigator__dropdown--show' : ''
+          }`}
+          aria-expanded={isOpen}
+        >
+          {frameworks.map((platform) => {
+            const title = platform.title;
+            const current = title === currentPlatform;
+            return (
+              <li
+                className={`platform-navigator__dropdown__item ${
+                  current ? 'platform-navigator__dropdown__item--current' : ''
+                }`}
+                key={platform.title}
+              >
+                <Link
+                  href={platform.href}
+                  key={platform.title}
+                  aria-current={current}
+                  className={`platform-navigator__dropdown__link`}
+                >
```

Comment:
Does it make sense to use the `Link` from `next/link` instead so that we get some of the next.js benefits like the prefetching and "client-side" navigation between routes?

At the moment there is a flash of unstyled text when we navigate between the different platforms and I think that's because we're reloading the whole route from scratch?